FALSEHOODS.md

```diff
@@ -53,6 +60,14 @@ surprising how many false assumptions programmers continue to make about them.
     from within a subset of countries, such as the international 00800 numbers.
     Some may be dialable only if the caller is a subscriber to a particular
     telecom company.
+    
+1. **You can send a text message to any phone number**
```

Comment:
We should probably add the corollary: that "Only mobile phones can receive text messages" which is also not true.